guardian/anghammarad
# Add a Node client

## What does this change?

This PR adds a node client for Anghammarad. This allows Anghammarad to be more easily used by projects written with node.js. It follows the logic of the existing Scala client.

## How to test

The test suite can be run using `yarn test`. To test out the client end-to-end, create a new node project, import the client and use it to send a message. You can see this client in action in the [tag-janitor](https://github.com/guardian/tag-janitor) project.

## How can we measure success?

The new node client for Anghammarad can be successfully used by other projects.
